daprdocs/content/en/reference/api/state_api.md

```diff
@@ -26,13 +26,14 @@ spec:
     value: <VALUE>
 ```
 
-The ```metadata.name``` is the name of the state store.
+| Setting | Description |
+| ------- | ----------- |
+| `metadata.name` | The name of the state store. |
+| `spec/metadata` | An open key value pair metadata that allows a binding to define connection properties. |
 
-the ```spec/metadata``` section is an open key value pair metadata that allows a binding to define connection properties.
+Support for multiple state stores was added, starting with 0.4.0 release. This is a breaking change from previous releases, as the state APIs were changed to support this new scenario.
 
-Starting with 0.4.0 release, support for multiple state stores was added. This is a breaking change from previous releases as the state APIs were changed to support this new scenario.
-
-Please refer https://github.com/dapr/dapr/blob/master/docs/decision_records/api/API-008-multi-state-store-api-design.md for more details.
+Refer to [the decision records](https://github.com/dapr/dapr/blob/master/docs/decision_records/api/API-008-multi-state-store-api-design.md) for more details.
```

Comment:
This paragraph can now be removed. It is over 2 years ago.

Reply:
This change looks great! I added a couple of comments on paragraphs that can be removed. If you see anything referring to a version of Dapr we should limit these and anything before v1.0 (like this) can be removed, since this is very old docs that need a refresh.